docs/sources/administration/configuration.md

```diff
@@ -1197,15 +1197,15 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
 
 ## [alerting]
 
-For more information about the Alerting feature in Grafana, refer to [Alerts overview]({{< relref "../alerting/_index.md" >}}).
+For more information about the legacy dashboard alerting feature in Grafana, refer to [Alerts overview]({{< relref "../alerting/_index.md" >}}).
 
 ### enabled
 
-Set to `false` to [enable Grafana 8 alerting]({{<relref "#unified_alerting">}}) and to disable legacy alerting engine. Default is `true`.
+Set to `false` to [enable Grafana alerting]({{<relref "#unified_alerting">}}) and to disable legacy alerting engine. Default is `true`.
```

Comment:
I would remove `Default is true` as it does not reflect the current situation anymore. After PR https://github.com/grafana/grafana/pull/42200 is merged it is more complicated than it is used to be.

Now the default value is `undefined` for both legacy and Grafana 8 alerts. When both are undefined the status of the alerting is determined in runtime and it depends on whether the user used legacy alerts before.

Reply:
Done